karma.conf.js

```diff
@@ -74,6 +74,9 @@ module.exports = function(config) {
 							'imports?define=>false'
 						]
 					}
+				],
+				noParse: [
+					/\/sinon\.js/,
```

Comment:
I'll try removing it but I copied this from the `o-header` Karma config... It was also in the issue on Webpack that I found the solution to.